src/httprpc.cpp

```diff
@@ -119,14 +122,17 @@ static bool multiUserAuthorized(std::string strUserPass)
             std::string strHashFromPass = HexStr(hexvec);
 
             if (TimingResistantEqual(strHashFromPass, strHash)) {
+                if (vFields.size() > 3) {
+                    walletNameOut = vFields[3];
```

Comment:
In commit "RPC: Allow rpcauth configs to specify..."

Should update `-rpcauth` documentation to mention the new field.